app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap

```diff
@@ -24,7 +24,7 @@ exports[`Metadata should render a Geolocation viewer when the metadata is type g
 <React.Fragment>
   <dl
     className="metadata-type-geolocation "
-    key="Geolocation Label"
+    key="undefined_0"
```

Comment:
Something is fishy here: previously, the label was "Geolocation  label", but upon adding the $index to the value, it registers as "undefined_0".  Either the fixtures are wrong or there is a problem with the label namespacing.